packages/doc-site/src/components/Dial.md

```diff
@@ -0,0 +1,227 @@
+```jsx
+import { COMPARISON_OPERATOR, StatusIcon, StreamType, DataType } from '@synchro-charts/core';
+<div style={{ width: '100%', height: '300px' }}>
+  <Dial
+    widgetId="test-widget"
+    dataStream={{
+      id: 'car-speed-alarm',
+      name: 'Wind temperature',
+      data: [
+        {
+          x: new Date(2001, 0, 0).getTime(),
+          y: 900,
+        },
+      ],
+      detailedName: 'Medium',
+      resolution: 0,
+      dataType: DataType.NUMBER,
+    }}
+    size={{
+      fontSize: 60,
+      dialThickness: 36,
+      iconSize: 48,
+      labelSize: 32,
+      unitSize: 32,
+    }}  
+    viewport={{ yMin: 0, yMax: 2000, duration: 1000 }}
+  />
+</div>;
+```
+Dial visualizes the latest value from a data stream by emphasizing the property value.For data stream of no unit it will display the percentage by using the two `yMin` and `yMax`.
```

Comment:
missing space after "the property value", needs some editing.

Phrase "by using the two `yMin` and `yMax` is grammatically awkward. 

"When no unit is provided, the dial will display a percentage value relative to the `yMin` and `yMax` values. For example, if the value is equal to `yMax`, it will display `100%`. For a value equal to `yMin`, it will display `0%`.

Reply:
Done.